java/configuration/src/main/java/sleeper/configuration/properties/SleeperPropertyValues.java

```diff
@@ -34,8 +34,12 @@ default boolean getBoolean(T property) {
         return Boolean.parseBoolean(get(property));
     }
 
-    default int getInt(T property) {
-        return Integer.parseInt(get(property));
+    default Integer getInt(T property) {
+        try {
+            return Integer.parseInt(get(property));
+        } catch (NumberFormatException parseEx) {
+            return null;
```

Comment:
If the string is actually not a valid integer, I think that should be thrown as an exception. Please can we add a unit test for that case?

I think if we null check the string returned from get, we don't need to catch a NumberFormatException at all?